lightning/src/routing/scoring.rs

```diff
@@ -40,8 +41,11 @@
 //!     ..ProbabilisticScoringParameters::default()
 //! };
 //! let scorer = ProbabilisticScorer::new(params, &network_graph);
+//! let random_seed = [42u8; 32];
```

Comment:
Maybe comment out of the lines here with a `#` at the beginning.

Reply:
Probably could use a fixed value, too. No need for a `KeysManager`.

Reply:
Addressed both with https://github.com/lightningdevkit/rust-lightning/commit/386a0b553130173f576ecc89424fcd6791a41dcc.